central/complianceoperator/v2/pipelines/complianceoperatorremediations/pipeline.go

```diff
@@ -0,0 +1,89 @@
+package complianceoperatorremediations
```

Comment:
For consistency we should put this in `complianceoperatorremediatioonsv2`

Reply:
done